.travis.yml

```diff
@@ -69,39 +72,30 @@ env:
 # Install system dependencies, namely a very barebones ROS setup.
 before_install:
   - sudo sh -c "echo \"deb http://packages.ros.org/ros/ubuntu $ROS_CI_DESKTOP main\" > /etc/apt/sources.list.d/ros-latest.list"
-  - sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-key 421C365BD9FF1F717815A3895523BAEEB01FA116
-  - sudo apt-get update -qq
-  - sudo apt-get install -y python-catkin-pkg python-rosdep python-wstool ros-$ROS_DISTRO-catkin
+  - wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
+  - sudo rm /var/lib/dpkg/lock
+  - sudo dpkg --configure -a
+  - sudo apt-get update
+  - sudo apt-get install ros-$ROS_DISTRO-desktop-full python-catkin-pkg python-wstool python-catkin-tools
   - source /opt/ros/$ROS_DISTRO/setup.bash
   # Prepare rosdep to install dependencies.
   - sudo rosdep init
   - rosdep update
+  - source /opt/ros/$ROS_DISTRO/setup.bash
 
 # Create a catkin workspace with the package under integration.
 install:
   - mkdir -p ~/catkin_ws/src
   - cd ~/catkin_ws/src
   - catkin_init_workspace
+  - git clone https://github.com/felixduvallet/pocketsphinx.git
   # Create the devel/setup.bash (run catkin_make with an empty workspace) and
   # source it to set the path variables.
   - cd ~/catkin_ws
   - catkin_make
-  - source devel/setup.bash
   # Add the package under integration to the workspace using a symlink.
-  - cd ~/catkin_ws/src
-  - ln -s $CI_SOURCE_PATH .
-
-# Install all dependencies, using wstool first and rosdep second.
-# wstool looks for a ROSINSTALL_FILE defined in the environment variables.
-before_script:
-  # source dependencies: install using wstool.
-  - cd ~/catkin_ws/src
-  - wstool init
-  - if [[ -f $ROSINSTALL_FILE ]] ; then wstool merge $ROSINSTALL_FILE ; fi
-  - wstool up
-  # package depdencies: install using rosdep.
-  - cd ~/catkin_ws
-  - rosdep install -y --from-paths src --ignore-src --rosdistro $ROS_DISTRO
+  - echo "source ~/catkin_ws/devel/setup.bash" >> ~/.bashrc
+  - source ~/.bashrc
 
```

Comment:
Please keep the ROSINSTALL file and all the dependency management that goes into the before_script.

Reply:
Ok!!